[4.x] `KafkaSeTest` failing on Windows

## Environment Details
* Helidon Version: 4.x
* Helidon SE or Helidon MP
* JDK version: 21
* OS: Windows
* Docker version (if applicable):

----------

## Problem Description

Test is failing.

## Steps to reproduce

run the test

```bash
cd tests/integration/kafka
mvn test
...
[ERROR] The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was cmd.exe /X /C ""C:\Program Files\Java\jdk-21\bin\java" -Xmx1024m -Dfile.encoding=UTF-8 -jar C:\Users\thiba\AppData\Local\Temp\surefire18277371614378420759\surefirebooter-20240124152425305_3.jar C:\Users\thiba\AppData\Local\Temp\surefire18277371614378420759 2024-01-24T15-24-25_200-jvmRun1 sur
efire-20240124152425305_1tmp surefire_0-20240124152425305_2tmp"
[ERROR] Error occurred in starting fork, check output in log
[ERROR] Process Exit Code: 1
[ERROR] Crashed tests:
[ERROR] io.helidon.messaging.connectors.kafka.KafkaSeTest
[ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was cmd.exe /X /C ""C:\Program Files\Java\jdk-21\bin\java" -Xmx1024m -Dfile.encoding=UTF-8 -jar C:\Users\thiba\AppData\Local\Temp\surefire18277371614378420759\surefirebooter-20240124152425305_3.jar C:\Users\thiba\AppData\Local\Temp\surefire18277371614378420759 2024-01-24T15-24-25_200-jvmRun1 sur
efire-20240124152425305_1tmp surefire_0-20240124152425305_2tmp"
[ERROR] Error occurred in starting fork, check output in log
[ERROR] Process Exit Code: 1
[ERROR] Crashed tests:
[ERROR] io.helidon.messaging.connectors.kafka.KafkaSeTest
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:643)
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:285)
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:250)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeProvider(AbstractSurefireMojo.java:1203)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:1055)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:871)
[ERROR]         at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:126)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:328)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:316)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:174)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.access$000(MojoExecutor.java:75)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor$1.run(MojoExecutor.java:162)
[ERROR]         at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute(DefaultMojosExecutionStrategy.java:39)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:159)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:105)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:73)
[ERROR]         at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:53)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:118)
[ERROR]         at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:261)
[ERROR]         at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:173)
[ERROR]         at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:101)
[ERROR]         at org.apache.maven.cli.MavenCli.execute(MavenCli.java:906)
[ERROR]         at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:283)
[ERROR]         at org.apache.maven.cli.MavenCli.main(MavenCli.java:206)
[ERROR]         at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[ERROR]         at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:283)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:226)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:407)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:348)
```

--- a/tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java
+++ b/tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2023 Oracle and/or its affiliates.
+ * Copyright (c) 2020, 2024 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -47,6 +47,7 @@ import io.helidon.messaging.Messaging;
 import io.helidon.messaging.connectors.kafka.AbstractSampleBean.Channel6;
 import io.helidon.messaging.connectors.kafka.AbstractSampleBean.Channel8;
 
+import org.apache.kafka.clients.admin.TopicListing;
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 import org.apache.kafka.common.header.Header;
 import org.apache.kafka.common.serialization.IntegerDeserializer;
@@ -74,6 +75,7 @@ public class KafkaSeTest extends AbstractKafkaTest {
 
     private static final Duration TIMEOUT = Duration.of(45, ChronoUnit.SECONDS);
     private static final String TEST_DQL_TOPIC = "test-dlq-topic";
+    private static final String TEST_DQL_TOPIC_1 = "test-dlq-topic-1";
     private static final String TEST_SE_TOPIC_1 = "special-se-topic-1";
     private static final String TEST_SE_TOPIC_2 = "special-se-topic-2";
     private static final String TEST_SE_TOPIC_3 = "special-se-topic-3";
@@ -128,6 +130,10 @@ public class KafkaSeTest extends AbstractKafkaTest {
 
     @AfterAll
     static void afterAll() {
+        List<String> topics = kafkaResource.getKafkaTestUtils().getTopics().stream()
+                .map(TopicListing::name)
+                .collect(Collectors.toList());
+        ADMIN.get().deleteTopics(topics);
         nackHandlerLogLogger.removeHandler(testHandler);
         kafkaResource.stopKafka();
     }
@@ -553,125 +559,113 @@ public class KafkaSeTest extends AbstractKafkaTest {
     @Test
     void consumeKafkaDLQNackExplicitConf() {
         kafkaResource.getKafkaTestUtils().createTopic(TEST_DQL_TOPIC, 2, (short) 1);
-        try {
-            List<String> result = consumerWithNack(KafkaConnector.configBuilder()
-                            .property("nack-dlq.topic", TEST_DQL_TOPIC)
-                            .bootstrapServers(KAFKA_SERVER)
-                            .groupId("test-group")
-                            .topic(TEST_SE_TOPIC_8)
-                            .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
-                            .enableAutoCommit(false)
-                            .keyDeserializer(StringDeserializer.class)
-                            .valueDeserializer(StringDeserializer.class)
-                            .build(),
-                    TEST_SE_TOPIC_8,
-                    "10"
-            );
 
-            assertThat(result, containsInAnyOrder("0", "1", "2", "4", "5", "6", "7", "8", "9", "10"));
+        List<String> result = consumerWithNack(KafkaConnector.configBuilder()
+                        .property("nack-dlq.topic", TEST_DQL_TOPIC)
+                        .bootstrapServers(KAFKA_SERVER)
+                        .groupId("test-group")
+                        .topic(TEST_SE_TOPIC_8)
+                        .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
+                        .enableAutoCommit(false)
+                        .keyDeserializer(StringDeserializer.class)
+                        .valueDeserializer(StringDeserializer.class)
+                        .build(),
+                TEST_SE_TOPIC_8,
+                "10"
+        );
 
-            List<ConsumerRecord<String, String>> dlqRecords = kafkaResource.consume(TEST_DQL_TOPIC);
+        assertThat(result, containsInAnyOrder("0", "1", "2", "4", "5", "6", "7", "8", "9", "10"));
 
-            assertThat(dlqRecords.size(), is(1));
-            ConsumerRecord<String, String> consumerRecord = dlqRecords.get(0);
-            Map<String, String> headersMap = Arrays.stream(consumerRecord.headers().toArray())
-                    .collect(Collectors.toMap(Header::key, h -> new String(h.value())));
-            assertThat(consumerRecord.key(), is("3"));
-            assertThat(consumerRecord.value(), is("3"));
-            assertThat(headersMap.get("dlq-error"), is("java.lang.Exception"));
-            assertThat(headersMap.get("dlq-error-msg"), is("BOOM!"));
-            assertThat(headersMap.get("dlq-orig-topic"), is(TEST_SE_TOPIC_8));
-            assertThat(headersMap.get("dlq-orig-offset"), is("3"));
-            assertThat(headersMap.get("dlq-orig-partition"), is("0"));
-        } finally {
-            ADMIN.get().deleteTopics(List.of(TEST_DQL_TOPIC));
-        }
+        List<ConsumerRecord<String, String>> dlqRecords = kafkaResource.consume(TEST_DQL_TOPIC);
+
+        assertThat(dlqRecords.size(), is(1));
+        ConsumerRecord<String, String> consumerRecord = dlqRecords.get(0);
+        Map<String, String> headersMap = Arrays.stream(consumerRecord.headers().toArray())
+                .collect(Collectors.toMap(Header::key, h -> new String(h.value())));
+        assertThat(consumerRecord.key(), is("3"));
+        assertThat(consumerRecord.value(), is("3"));
+        assertThat(headersMap.get("dlq-error"), is("java.lang.Exception"));
+        assertThat(headersMap.get("dlq-error-msg"), is("BOOM!"));
+        assertThat(headersMap.get("dlq-orig-topic"), is(TEST_SE_TOPIC_8));
+        assertThat(headersMap.get("dlq-orig-offset"), is("3"));
+        assertThat(headersMap.get("dlq-orig-partition"), is("0"));
     }
 
     @Test
     void consumeKafkaDLQNackDerivedConf() {
-        kafkaResource.getKafkaTestUtils().createTopic(TEST_DQL_TOPIC, 2, (short) 1);
-        try {
-            List<String> result = consumerWithNack(KafkaConnector.configBuilder()
-                            .property("nack-dlq", TEST_DQL_TOPIC)
-                            .bootstrapServers(KAFKA_SERVER)
-                            .groupId("test-group")
-                            .topic(TEST_SE_TOPIC_9)
-                            .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
-                            .enableAutoCommit(false)
-                            .keyDeserializer(StringDeserializer.class)
-                            .valueDeserializer(StringDeserializer.class)
-                            .build(),
-                    TEST_SE_TOPIC_9,
-                    "10"
-            );
+        kafkaResource.getKafkaTestUtils().createTopic(TEST_DQL_TOPIC_1, 2, (short) 1);
 
-            assertThat(result, containsInAnyOrder("0", "1", "2", "4", "5", "6", "7", "8", "9", "10"));
+        List<String> result = consumerWithNack(KafkaConnector.configBuilder()
+                        .property("nack-dlq", TEST_DQL_TOPIC_1)
+                        .bootstrapServers(KAFKA_SERVER)
+                        .groupId("test-group")
+                        .topic(TEST_SE_TOPIC_9)
+                        .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
+                        .enableAutoCommit(false)
+                        .keyDeserializer(StringDeserializer.class)
+                        .valueDeserializer(StringDeserializer.class)
+                        .build(),
+                TEST_SE_TOPIC_9,
+                "10"
+        );
 
-            List<ConsumerRecord<String, String>> dlqRecords = kafkaResource.consume(TEST_DQL_TOPIC);
+        assertThat(result, containsInAnyOrder("0", "1", "2", "4", "5", "6", "7", "8", "9", "10"));
 
-            assertThat(dlqRecords.size(), is(1));
-            ConsumerRecord<String, String> consumerRecord = dlqRecords.get(0);
-            Map<String, String> headersMap = Arrays.stream(consumerRecord.headers().toArray())
-                    .collect(Collectors.toMap(Header::key, h -> new String(h.value())));
-            assertThat(consumerRecord.key(), is("3"));
-            assertThat(consumerRecord.value(), is("3"));
-            assertThat(headersMap.get("dlq-error"), is("java.lang.Exception"));
-            assertThat(headersMap.get("dlq-error-msg"), is("BOOM!"));
-            assertThat(headersMap.get("dlq-orig-topic"), is(TEST_SE_TOPIC_9));
-            assertThat(headersMap.get("dlq-orig-offset"), is("3"));
-            assertThat(headersMap.get("dlq-orig-partition"), is("0"));
-        } finally {
-            ADMIN.get().deleteTopics(List.of(TEST_DQL_TOPIC));
-        }
+        List<ConsumerRecord<String, String>> dlqRecords = kafkaResource.consume(TEST_DQL_TOPIC_1);
+
+        assertThat(dlqRecords.size(), is(1));
+        ConsumerRecord<String, String> consumerRecord = dlqRecords.get(0);
+        Map<String, String> headersMap = Arrays.stream(consumerRecord.headers().toArray())
+                .collect(Collectors.toMap(Header::key, h -> new String(h.value())));
+        assertThat(consumerRecord.key(), is("3"));
+        assertThat(consumerRecord.value(), is("3"));
+        assertThat(headersMap.get("dlq-error"), is("java.lang.Exception"));
+        assertThat(headersMap.get("dlq-error-msg"), is("BOOM!"));
+        assertThat(headersMap.get("dlq-orig-topic"), is(TEST_SE_TOPIC_9));
+        assertThat(headersMap.get("dlq-orig-offset"), is("3"));
+        assertThat(headersMap.get("dlq-orig-partition"), is("0"));
     }
 
     @Test
     void consumeKafkaKillChannelNack() {
         var testTopic = "test-topic";
         kafkaResource.getKafkaTestUtils().createTopic(testTopic, 2, (short) 1);
-        try {
-            List<String> result = consumerWithNack(KafkaConnector.configBuilder()
-                            .bootstrapServers(KAFKA_SERVER)
-                            .groupId("test-group")
-                            .topic(testTopic)
-                            .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
-                            .enableAutoCommit(false)
-                            .keyDeserializer(StringDeserializer.class)
-                            .valueDeserializer(StringDeserializer.class)
-                            .build(),
-                    testTopic,
-                    null //  wait for channel being killed
-            );
-            assertThat(result, contains("0", "1", "2"));
-        } finally {
-            ADMIN.get().deleteTopics(List.of(testTopic));
-        }
+
+        List<String> result = consumerWithNack(KafkaConnector.configBuilder()
+                        .bootstrapServers(KAFKA_SERVER)
+                        .groupId("test-group")
+                        .topic(testTopic)
+                        .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
+                        .enableAutoCommit(false)
+                        .keyDeserializer(StringDeserializer.class)
+                        .valueDeserializer(StringDeserializer.class)
+                        .build(),
+                testTopic,
+                null //  wait for channel being killed
+        );
+        assertThat(result, contains("0", "1", "2"));
     }
 
     @Test
     void consumeKafkaLogOnlyNack() {
-        var testTopic = "test-topic";
+        var testTopic = "test-topic-1";
         kafkaResource.getKafkaTestUtils().createTopic(testTopic, 2, (short) 1);
-        try {
-            List<String> result = consumerWithNack(KafkaConnector.configBuilder()
-                            .bootstrapServers(KAFKA_SERVER)
-                            .property("nack-log-only", "true")
-                            .groupId("test-group")
-                            .topic(testTopic)
-                            .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
-                            .enableAutoCommit(false)
-                            .keyDeserializer(StringDeserializer.class)
-                            .valueDeserializer(StringDeserializer.class)
-                            .build(),
-                    testTopic,
-                    "10"
-            );
-            assertThat(result, containsInAnyOrder("0", "1", "2", "4", "5", "6", "7", "8", "9", "10"));
-            assertThat(logNackHandlerWarnings, hasItem("NACKED Message - ignored key: 3 topic: test-topic offset: 3 partition: 0"));
-        } finally {
-            ADMIN.get().deleteTopics(List.of(testTopic));
-        }
+
+        List<String> result = consumerWithNack(KafkaConnector.configBuilder()
+                        .bootstrapServers(KAFKA_SERVER)
+                        .property("nack-log-only", "true")
+                        .groupId("test-group")
+                        .topic(testTopic)
+                        .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
+                        .enableAutoCommit(false)
+                        .keyDeserializer(StringDeserializer.class)
+                        .valueDeserializer(StringDeserializer.class)
+                        .build(),
+                testTopic,
+                "10"
+        );
+        assertThat(result, containsInAnyOrder("0", "1", "2", "4", "5", "6", "7", "8", "9", "10"));
+        assertThat(logNackHandlerWarnings, hasItem("NACKED Message - ignored key: 3 topic: test-topic-1 offset: 3 partition: 0"));
     }
 
     List<String> consumerWithNack(Config c, String topic, String lastExpectedValue) {

